package.json

```diff
@@ -82,6 +86,7 @@
     "build": "pnpm build-package _internal && pnpm build-package core && pnpm build-package infinite && pnpm build-package immutable && pnpm build-package mutation && pnpm build-package subscription",
     "build:e2e": "pnpm next build e2e/site",
     "build-package": "bunchee --cwd",
+    "attw": "attw --pack",
```

Comment:
what does `attw` do here? seems not in the dependency

Reply:
https://github.com/arethetypeswrong/arethetypeswrong.github.io


It's like publint, but for types